helix-term/src/commands.rs

```diff
@@ -5392,7 +5395,32 @@ fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
             textobject(cx.editor);
             cx.editor.last_motion = Some(Motion(Box::new(textobject)));
         }
-    })
+    });
+
+    if let Some((title, abbrev)) = match objtype {
+        textobject::TextObject::Inside => Some(("Match inside", "mi")),
+        textobject::TextObject::Around => Some(("Match around", "ma")),
+        textobject::TextObject::Movement => None,
```

Comment:
This should be unreachable.

Reply:
If you mean the movement arm, yes, but rust wants me to handle the enum exhaustively so I obliged.

Reply:
There's an `unreachable!()`  macro that can be used in instances like these, so this block would turn into:

```rust
let (title, abbrev) = match objtype {
    textobject::TextObject::Inside => ("Match inside", "mi"),
    textobject::TextObject::Around => ("Match around", "ma"),
    textobject::TextObject::Movement => unreachable!(),
}
```

`unreachable!()` actually panics, and since textobjects queries are read at runtime from query files it might be better to simply return from that point.

Reply:
So are we looking to panic here or no? I'm s little confused where we've landed. I was considering using a `_` match arm to swallow and ignore, or maybe just `log!`.

Reply:
I think we can simply do an early return here since there would be nothing to display in the infobox anyway:

```rust
let (title, abbrev) = match objtype {
    textobject::TextObject::Inside => ("Match inside", "mi"),
    textobject::TextObject::Around => ("Match around", "ma"),
    textobject::TextObject::Movement => return,
}

Reply:
Resolved in latest